src/Bielu.Examine.Umbraco/ElasticsearchExamineIndexRebuilder.cs

```diff
@@ -164,10 +168,22 @@ private void RebuildIndex(string indexName, TimeSpan delay, CancellationToken ca
 
                     populator.Populate(index);
                 }
+
+                // Reset the cache for the ExamineManagementController
+                var cacheKey = "temp_indexing_op_" + indexName;
+                _runtimeCache.Insert(cacheKey, () => "tempValue", TimeSpan.FromMinutes(5));
+
                 if (index is IBieluExamineIndex elasticIndex)
                 {
                     elasticIndex.SwapIndex();
                 }
+
+                // Wait 1 second to get the right DocumentCount back from Elastic
+                Thread.Sleep(1000);
```

Comment:
not sure abotu thread sleep, i am sure we can find better approach for this.

Reply:
You're right, I didn't like this construction was well, but it also turned out that it could be solved much more simple by setting the cache timeout to 1 second.

Reply:
I will go through this pr on Saturday as need have think about few things

Reply:
I accidentally made another commit in this PR that I actually wanted to place in a new PR to fix a problem with the Forms indexation. I'm a bit of a bungler when it comes to git :-(